.github/workflows/usermods.yml

```
@@ -0,0 +1,71 @@
+
+  build:
+    name: Build Enviornments
+    runs-on: ubuntu-latest
+    needs: get_usermod_envs
+    strategy:
+      fail-fast: false
+      matrix:
+        usermod: ${{ fromJSON(needs.get_usermod_envs.outputs.usermods) }}
+        environment: [usermods_esp32, usermods_esp32c3, usermods_esp32s2, usermods_esp32s3]
+    steps:
+    - uses: actions/checkout@v4
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)